### PR TITLE
Add Flatfile

### PR DESCRIPTION
**Pull request by fmerian**
https://flatfile.com/

## checklist

Please make sure you've complied with the [contribution guidelines](https://github.com/agamm/awesome-developer-first/blob/main/CONTRIBUTING.md):
- [x] The homepage clearly states that the product is for developers.
  > "Headless", "API-First", "SaaS" are frequently used keywords.
  > Usually this means that the front page has some code examples :)
- [x] The product is available now and requires payment.
- [x] The PR has a descriptive title -- e.g., `Add {Product}`, not `Update README.md`.
- [x] The new submission isn't a duplicate.
- [x] The entry has a description, is sorted alphabetically, and ends with a full stop.

Thanks!

### CODE REVIEW

**Comment by agamm** [comment]
I would use the words "Data import" somewhere there.

**Comment by fmerian** [comment]
@agamm better?

File: README.md [hunk 1]
```diff
@@ -242,6 +242,7 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 * [APITemplate.io](https://apitemplate.io/) - APIs to generate images and PDFs from reusable templates.
 * [Cloudinary](https://cloudinary.com/) - Optimize, create, manage, and deliver images and videos.
 * [Dyte](https://dyte.io) - Build powerful live experiences in your product with an all-in-one Video SDK.
+* [Flatfile](https://flatfile.com/) - Programmatically import API-ready data from files.
```
Comment (agamm) [inline README.md:245]:
```suggestion
* [Flatfile](https://flatfile.com/) - Programmatically import data from files so it's API-ready.
```

**Review by agamm — APPROVED** [review]
Slightly changed.

**Pull request merged**